### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by Lu191**
Fixed the following issues:
- Duplicate alerts for tendermint custom tasks
- Conversion issues in the Tendermint custom task `TaskTendermintBlockMissed`
- Bug in the Tendermint custom task `TaskTendermintPositionChanged` when the validators in the active set are more than 200
- Output of the message when the position has changed (it was counterintuitive)
- Missing import in the task `TaskAutoUpdater`

### CODE REVIEW

File: srvcheck/chains/substrate.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ class Substrate (Chain):
 	BLOCKTIME = 15 
 	EP = 'http://localhost:9933/'
 
-	def __init__(self, conf):
+	def __init__(self, conf, addTasks=False):
```
Comment (dakk) [inline srvcheck/chains/substrate.py:46]: please explain this new field

File: srvcheck/chains/tendermint.py [hunk 2]
```diff
@@ -20,13 +20,13 @@ def run(self):
 
 		if not self.prev:
 			self.prev = nblockh
-		elif nblockh - self.prev >= self.BLOCK_WINDOW:
-			block = self.prev
+		elif int(nblockh) - (self.prev) >= int(self.BLOCK_WINDOW):
```
Comment (dakk) [inline srvcheck/chains/tendermint.py:23]: move all casts in this method to __init__ and getHeight()

**Review by dakk — CHANGES_REQUESTED** [review]

File: srvcheck/chains/substrate.py (hunk 1 above)
Comment (Lu191) [reply srvcheck/chains/substrate.py:46]: The problem was that the `__init__ `was called twice, when we try to detect the chain `x.detect(config)` and when we initialize the chain with `chain = x(config)`, in this way the custom tasks would have been added twice, with this new param we can fix this issue, specifying when the `__init__ `should add the tasks and when not, in this case, I decided to add the custom tasks only when we initialize the chain, so `chain = x(config, True) `

Comment (Lu191) [reply srvcheck/chains/substrate.py:46]: Let's do the second solution

Comment (dakk) [reply srvcheck/chains/substrate.py]: I mean, something like this:

```python
class Tendermin:
   TAKS = [ TaskPincoPalla, TaskGinoPino ]
```

Comment (Lu191) [reply srvcheck/chains/substrate.py]: Done

**Review by dakk — APPROVED** [review]

**Pull request merged**